src/convertToObject.js

```diff
@@ -6,7 +6,28 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const declarations = sourceString.trim().split(';');
+
+  const stylesObj = {};
+
+  declarations.forEach((declaration) => {
+    const parts = declaration.split(':');
```

Comment:
```suggestion
    const [property, value] = declaration.split(':');
```
You can use destructing